scripts/solc_abigen.sh

```diff
@@ -27,7 +27,7 @@ solFiles=(
 )
 
 dld_solc() {
-  curl -L "https://binaries.soliditylang.org/linux-amd64/solc-linux-amd64-${SOLC_VER}" -o solc && chmod +x solc
+  curl -L "https://github.com/ethereum/solidity/releases/download/${SOLC_VER}/solc-static-linux" -o solc && chmod +x solc
```

Comment:
Why this change? I thought binaries.soliditylang.org is the recommended URL.

Reply:
updated